components/brave_wallet/browser/brave_wallet_p3a.cc

```diff
@@ -0,0 +1,83 @@
+/* Copyright 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include <string>
+
+#include "base/metrics/histogram_macros.h"
+#include "brave/components/brave_wallet/browser/brave_wallet_p3a.h"
+#include "brave/components/brave_wallet/browser/brave_wallet_service.h"
+#include "brave/components/brave_wallet/browser/keyring_service.h"
+#include "brave/components/brave_wallet/browser/pref_names.h"
+#include "components/prefs/pref_service.h"
+
+namespace brave_wallet {
+
+// Has the Wallet keyring been created?
+// i) Yes, ii) No
```

Comment:
Are these in the correct order? I'd expect casting a bool would result in the `0` value for false.

Reply:
Nice catch, I will update the comment

Reply:
https://github.com/brave/brave-core/pull/11929/commits/eb2a3a6dc9ae408c0d75f6512c5e0298b57153fa